config/zephyr/Kconfig

```diff
@@ -225,6 +225,7 @@ config APP_LINK_WITH_CHIP
 
 config CHIP_OTA_IMAGE_BUILD
 	bool "Generate OTA image"
+	select DFU_MULTI_IMAGE_PACKAGE_BUILD
```

Comment:
Isn't this `DFU_MULTI_IMAGE_PACKAGE_BUILD` something nrfconnect specific? I think we should not select it for the whole Zephyr platform, right?

Reply:
Done